app/src/main/java/com/mathewsachin/fategrandautomata/accessibility/ScriptRunnerService.kt

```diff
@@ -175,12 +179,21 @@ class ScriptRunnerService : AccessibilityService() {
 
     override fun onInterrupt() {}
 
-    private var foregroundAppName = ""
+    private var currentFgoServer: GameServerEnum? = null
 
+    /**
+     * This method is called on any [AccessibilityEvent].
```

Comment:
You may want to write here that it's not for every AccessibilityEvent but only for once we've subscibed in `@xml/script_runner_service`.
Right now we've only subscribed for `TYPE_WINDOW_STATE_CHANGED`.

Reply:
Done